src/sentry/static/sentry/app/components/activity/note.jsx

```diff
@@ -6,7 +6,52 @@ import ConfigStore from '../../stores/configStore';
 import LinkWithConfirmation from '../../components/linkWithConfirmation';
 import {t} from '../../locale';
 
+function isSafeHref(href) {
+  try {
+    return /^https?:\/\/./i.test(decodeURIComponent(unescape(href)));
+  } catch(e) {
+    return false;
+  }
+}
+
+
+// We need to implement our own marked Renderer to not render
+// potentially malicious uris.
+// This is copy/pasted from
+// https://github.com/chjj/marked/blob/master/lib/marked.js#L869-L888
+// and modified.
+function Renderer(options) {
+  this.options = options || {};
+}
+
+Object.assign(Renderer.prototype, marked.Renderer.prototype);
+
+Renderer.prototype.link = function(href, title, text) {
+  // For a bad link, just return the plain text href
+  if (this.options.sanitize && !isSafeHref(href)) return href;
```

Comment:
> Can we really make this assumption that returning href is safe?

It's just plain text, it doesn't get rendered as DOM or anything, so yes. And since this is making things more restrictive, I'd rather opt to returning the original value rather than silently removing it so people are less confused.

> return marked.Renderer.prototype.apply(this, arguments);

Because this has older broken logic that we're replacing. Also, pretty sure `marked` is unmaintained. It hasn't been touched in a year.